includes/blocks/list-block/render.php

```diff
@@ -237,15 +238,13 @@ function benenson_render_list_item( $data, $display_excerpt = 0, $cta_text = ''
 				?>
 				</p>
 			<?php endif; ?>
-			<?php if ( ! empty( $data['buttonText'] ) && ! empty( $data['buttonLink'] ) ) : ?>
```

Comment:
What happened to the `buttonText` and `buttonLink` data—were they unused—I can't tell from these changes?

Reply:
I thought they were unused, but i've just put them back in because they could be a situation where you need both.